frontend/src/app/main/ui/workspace/context_menu.cljs

```diff
@@ -342,7 +342,7 @@
 
         ;; components can't be ungrouped
         has-frame? (->> shapes (d/seek #(and (cfh/frame-shape? %) (not (ctk/instance-head? %)) (not (ctk/is-variant-container? %)))))
-        has-group? (->> shapes (d/seek #(and (cfh/group-shape? %) (not (ctk/instance-head? %)) (not (ctk/is-variant-container? %)))))
+        has-group? (->> shapes (d/seek #(and (cfh/group-shape? (get objects %)) (not (ctk/instance-head? %)) (not (ctk/is-variant-container? %)))))
```

Comment:
I don't think this is the ideal solution, group-shape? is ready to receive two parameters (objects contains all the page elements), probably the problem is what's in the shapes variable